libsolidity/interface/CompilerStack.cpp

```diff
@@ -509,22 +509,31 @@ string CompilerStack::applyRemapping(string const& _path, string const& _context
 	};
 
 	size_t longestPrefix = 0;
+	size_t longestContext = 0;
 	string longestPrefixTarget;
+
 	for (auto const& redir: m_remappings)
 	{
-		// Skip if we already have a closer match.
-		if (longestPrefix > 0 && redir.prefix.length() <= longestPrefix)
+		string contextFileString = sanitizePath(redir.context);
```

Comment:
Why not just name this `context` and the one below `prefix` instead of `contextFileString` and `prefixFileString?